backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -2083,3 +2083,13 @@ databaseChangeLog:
         - dropUniqueConstraint:
             tableName: organization
             constraintName: uk__facility__name
+      id: add-email-back-to-api-user-view
+      author: josh@skylight.digital
+      comment: Add a column for email address to api_user_no_phi_view
+      changes:
```

Comment:
The automatic rollback for this would be to drop the view entirely. I think that’s honestly OK in this case, but the alternative is to either recreate the previous view explicitly or refer back to the migration that created it. (I think that changeset  creates too many other things to be a safe rollback, though.)

Reply:
I actually forgot about the rollback in this case, so good catch. I just added a commit to explicitly drop the new view and recreate the prior one.